src/runtime-rs/crates/hypervisor/src/qemu/network.rs

```diff
@@ -5,12 +5,15 @@
 
 use std::collections::HashMap;
 use std::convert::TryFrom;
+use std::fs::{File, OpenOptions};
 use std::os::fd::RawFd;
 
 use crate::utils::clear_fd_flags;
 use crate::{Address, NetworkConfig};
 use anyhow::{anyhow, Context, Result};
 
+use dbs_utils::net::Tap;
```

Comment:
This breaks build on s390x as reported by https://github.com/kata-containers/kata-containers/actions/runs/8188007403/job/22442402744?pr=8866 :

```
error[E0433]: failed to resolve: use of undeclared crate or module `dbs_utils`
  --> crates/hypervisor/src/qemu/network.rs:15:5
   |
15 | use dbs_utils::net::Tap;
   |     ^^^^^^^^^ use of undeclared crate or module `dbs_utils`
```

Reply:
Yes, it's actually questionable to pull in an additional dependency when the same functionality seems to be available already in the `resource` crate.  I meant to address this in a subsequent PR but if it's breaking a build then it should perhaps be addressed earlier. (?)

Reply:
> Yes, it's actually questionable to pull in an additional dependency when the same functionality seems to be available already in the `resource` crate. I meant to address this in a subsequent PR but if it's breaking a build then it should perhaps be addressed earlier. (?)

Required checks depend on this build to succeed. The build break must be addressed in this PR.

Reply:
@pmores @gkurz I have checked the code and found that the dbs_utils dependencies `dbs-utils = { path = "../../../dragonball/src/dbs_utils" }` was moved to [`target.'cfg(not(target_arch = "s390x"))'.dependencies]` and hidden which causes the error of `undeclared crate or module` in s390x platform. And IMO, it should not be hidden now because `dbs_utils` is just a public utils for all vmms